rllib/models/torch/fcnet.py

```diff
@@ -21,39 +21,76 @@ def __init__(self, obs_space, action_space, num_outputs, model_config,
                               model_config, name)
         nn.Module.__init__(self)
 
+        activation = get_activation_fn(
+            model_config.get("fcnet_activation"), framework="torch")
         hiddens = model_config.get("fcnet_hiddens")
-        activation = _get_activation_fn(model_config.get("fcnet_activation"))
+        no_final_linear = model_config.get("no_final_linear")
+
+        # TODO(sven): implement case: vf_shared_layers = False.
+        # vf_share_layers = model_config.get("vf_share_layers")
```

Comment:
Should we raise an error if this is set?

Reply:
done